### PR TITLE
Add Run Issues to the Test Analytics docs

### PR DESCRIPTION
**Pull request by meghan-kradolfer**
Adding the new Run Issues into the Test Analytic docs. Including a useful description on each issue that we find and where the user can see the issues appear.

**Changes:**
- Added run issues to 'Configuring test suites' 
- Included a private beta notice which will be [removed when we go GA (29/9)](https://linear.app/buildkite/issue/PIE-1996/remove-private-beta-notice-from-docs)
- Added run issues to the TA overview page
- Removed icons on the overview page (request from Jasmin)

**Linear ticket:** 
[Add Run Issues to the Test Analytics docs](https://linear.app/buildkite/issue/PIE-1974/add-issues-page-to-docs)

Screenshot 2023-09-19 at 10 12 44 AM
Screenshot 2023-09-19 at 10 11 20 AM

### CODE REVIEW

File: data/tiles.yml [hunk 1]
```diff
@@ -2,12 +2,13 @@
 test_analytics_features:
   - title: "Deep performance analysis"
     url: "/docs/test-analytics/test-suites#trends-and-analysis"
-    image_path: "images/test_analytics/icon-performance-analysis.svg"
```
Comment (KatieWright26) [inline data/tiles.yml:5]: should these images have been removed 🤔

Comment (meghan-kradolfer) [reply data/tiles.yml:5]: Yup, I mentioned in the description that the overview icons are being removed [from the request here ](https://linear.app/buildkite/issue/PIE-1974/add-issues-page-to-docs#comment-9a14d5f5)

File: pages/test_analytics/test_suites.md [hunk 2]
```diff
@@ -35,6 +35,25 @@ Alternatively, you can create [scheduled builds](/docs/pipelines/scheduled-build
 
 Test Analytics reviews the test results to detect flaky tests after every test run.
 
+## Run issues
+
+<%= image "run-issues.png", alt: "Screenshot of a run with issues displaying in a list, including flaky, slow and failures." %>
+
+Test Analytics will automatically detect issues per run. For each test, we currently detect up to three issues: flaky, slow and failure, listed in the following order of urgency:
```
Comment (KatieWright26) [inline pages/test_analytics/test_suites.md:42]: Technically its "number of issues, flaky, slow, failed", I wonder if we should clarify that?

**Review by KatieWright26 — COMMENTED** [review]
I'm keen to know what the docs people think - I feel like the beta flag should be at the top of the section?
Also, the slow issue wording feels a bit awkward still; keen for feedback around that

File: data/tiles.yml [hunk 3]
```diff
@@ -2,12 +2,13 @@
...
     desc: "Automatic tracing across your test suite, deeply integrated with your programming language and test framework."
   - title: "Find and fix flaky tests"
     url: "/docs/test-analytics/test-suites#detecting-flaky-tests"
-    image_path: "images/test_analytics/icon-find-flaky-tests.svg"
     desc: "Quickly identify which tests are the most disruptive for your team, and get a head-start on fixing them."
+  - title: "See urgent issues in your run"
+    url: "/docs/test-analytics/test-suites#run-issues"
+    desc: "Find problematic tests in a given run and resolve their issues."
```
Comment (MelissaKaulfuss) [inline data/tiles.yml:11]: If this is factually correct then these words sounds better:

```suggestion
    desc: "Find and resolve issues with problematic tests in every run."
```

File: pages/test_analytics/test_suites.md (hunk 2 above)
Comment (MelissaKaulfuss) [inline pages/test_analytics/test_suites.md:42]:
```suggestion
Test Analytics will automatically detect issues for each run. For each test, we currently detect up to three issues: flaky, slow and failure, listed in the following order of urgency:
```

Comment (MelissaKaulfuss) [reply pages/test_analytics/test_suites.md:42]: @KatieWright26 do you mean that it should be?

>  For each test, we detect flaky, slow and failed issues, along with the total number of issues found. These issues are listed in the following order of urgency:

File: pages/test_analytics/test_suites.md [hunk 4]
```diff
@@ -35,6 +35,25 @@ Alternatively, you can create [scheduled builds](/docs/pipelines/scheduled-build
...
+
+- **Flaky:** [See section on detecting flaky tests](#detecting-flaky-tests).
+
+- **Slow:** Slowness is measured on the comparative performance of tests within the current run. A slow test is seen as an outlier when it is part of the slowest 1% of tests that took more than 15% of the total run time.
```
Comment (MelissaKaulfuss) [inline pages/test_analytics/test_suites.md:46]: This is haaard. Does this make sense still?

```suggestion
- **Slow:** Slowness is measured by the comparative performance of tests within the current run. A test is considered slow when it is in the slowest 1% of tests that took more than 15% of the total run time.
```

Comment (KatieWright26) [reply pages/test_analytics/test_suites.md:46]: It is right?? ChatGPT offered:
> When the slowest 1% of tests extend beyond 15% of the overall run time, our system takes charge. We automatically flag these slower tests, ensuring your testing process stays agile and efficient. 
That feels closer to what we want?

File: pages/test_analytics/test_suites.md (hunk 2 above)
Comment (meghan-kradolfer) [reply pages/test_analytics/test_suites.md]: The confusion here is around the ordering, we say it is ordered 'flaky, slow and failure' where it is actually ordered 'total number of issues, flaky, slow and failure`

Something like:
For each test, we currently detect up to three issues: flaky, slow and failure. These issues are organised and presented in the following order based on their level of urgency, tests with the most amount of issues will be positioned at the top of the list.

File: pages/test_analytics/test_suites.md (hunk 4 above)
Comment (MelissaKaulfuss) [reply pages/test_analytics/test_suites.md:49]: I think our human words are far better than ChatGPT's attempt.

How about?
> Slowness is measured by the comparative performance of tests within the current run. The system automatically flags slow tests when the slowest 1% of tests take more than 15% of the overall run time.

File: pages/test_analytics/test_suites.md (hunk 2 above)
Comment (MelissaKaulfuss) [reply pages/test_analytics/test_suites.md]: Still confused, almost there!

The system identifies these 3 issues for tests (the entire test file), and orders the list of test files with the one that has the most issues at the top, decending? eg.
**Worst spec**
`user_authentication_spec.rb`
* Total issues: 12
   * Slow: 4
   * Flaky: 4
   * Failed: 4

**Second worst spec**
`hello_user_spec.rb`
 * Total issues: 6
   * Slow: 2
   * Flaky: 2
   * Failed: 2

Comment (MelissaKaulfuss) [inline pages/test_analytics/test_suites.md]:
```suggestion
Test Analytics will automatically detect issues per run. For each test, we currently detect three issues: flaky, slow and failure. And display total number of issues, along with the following, ordered by the most problematic to least problematic:
```

File: pages/test_analytics/test_suites.md (hunk 4 above)
Comment (MelissaKaulfuss) [inline pages/test_analytics/test_suites.md:49]:
```suggestion
- **Slow:** Slowness is measured by the comparative performance of tests within the current run. The system automatically flags slow tests when the slowest 1% of tests take more than 15% of the overall run time.
```

File: pages/test_analytics/test_suites.md (hunk 2 above)
Comment (meghan-kradolfer) [reply pages/test_analytics/test_suites.md]: It is per execution within test file, so not the whole file. The issues with either have a count of `1 | 0` so like your example:

**Worst spec**
`user_authentication_spec.rb:21`
Total issues: 3
Slow: 1
Flaky: 1
Failed: 1

**Second worst spec**
`user_authentication_spec.rb:2`
Total issues: 2
Slow: 1
Flaky: 0
Failed: 1

Do you think we should include that example in the doc?

File: pages/test_analytics/test_suites.md [hunk 5]
```diff
@@ -35,6 +35,26 @@ Alternatively, you can create [scheduled builds](/docs/pipelines/scheduled-build
...
+> This feature is currently in private beta. If you would like to request access to Run issues, [contact us here](support@buildite.com)
+
+<%= image "run-issues.png", alt: "Screenshot of a run with issues displaying in a list, including flaky, slow and failures." %>
+
+Test Analytics will automatically detect issues for each run. For each test, we currently detect up to three issues: flaky, slow and failure, listed in the following order of urgency:
+
+- **Flaky:** [See section on detecting flaky tests](#detecting-flaky-tests).
+
+- **Slow:** Slowness is measured by the comparative performance of tests within the current run. The system automatically flags slow tests when the slowest 1% of tests take more than 15% of the overall run time. This overall percentage can be manually amended within the suite settings.
+
```
Comment (meghan-kradolfer) [inline pages/test_analytics/test_suites.md:50]: @MelissaKaulfuss @KatieWright26 new sentence has been added to the end of this to inform users that the overall percentage can be amended within settings

Comment (KatieWright26) [reply pages/test_analytics/test_suites.md:50]: LGTM!

**Review by KatieWright26 — APPROVED** [review]

**Review by MelissaKaulfuss — APPROVED** [review]
Looking good! ✅

**Pull request merged**